Sources/TuistSupport/Constants.swift

```diff
@@ -78,6 +78,7 @@ public enum Constants {
         public static let osLog = "TUIST_CONFIG_OS_LOG"
         /// `tuistBinaryPath` is used for specifying the exact tuist binary in tuist tasks.
         public static let tuistBinaryPath = "TUIST_CONFIG_BINARY_PATH"
+        public static let tuistBuildForDevelopment = "TUIST_BUILD_FOR_DEVELOPMENT"
```

Comment:
If possible would be great to avoid the raw generated projects have `TUIST` related env variables to keep them as portable and standalone as possible. Tuist commands may augment the generated projects as needed via mappers / Model APIs etc...